cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -585,6 +585,32 @@ impl xcm_executor::Config for XcmConfig {
 			ResolveAssetTo<StakingPot, crate::NativeAndAssets>,
 			AccountId,
 		>,
+		// This trader allows to pay with `is_sufficient=true` "Trust Backed" assets from dedicated
+		// `pallet_assets` instance - `Assets`.
+		cumulus_primitives_utility::TakeFirstAssetTrader<
```

Comment:
@joepetrowski to confirm - I don't see a strong reason for keeping `TakeFirstAssetTrader` for "Trust Backed Assets" either in testnets or prod runtimes.

Reply:
In prod we probably want 1 release where they overlap. We expect pools to form pretty quickly, but you could have:

- Upgrade that removes `TakeFirstAssetTrader`.
- Someone sends a message to pay fees with a sufficient asset.
- Pool gets established.

The message will not work. I'd just put `TakeFirstAssetTrader` at the end of the tuple so that once there is a pool, it will default to the pool exchange rate.

For testnets it's probably fine to just remove it.

Reply:
if there is no some new security concern on having these two take first traders (with Assets, and ForeignAssets), I do not see a reason to complicate things for a transition period. They are both now set in prod, lets keep them both for one release. And the swap trader appears first in the tuple and will be applied first as soon as there is a pool.